core/container/bit_array/bit_array.odin

```diff
@@ -157,14 +154,14 @@ get :: proc(ba: ^Bit_Array, #any_int index: uint, allocator := context.allocator
 }
 
 /*
-	In:
-		- ba:    ^Bit_Array - a pointer to the Bit Array
-		- index: The bit index. Can be an enum member.
+In: <br>
```

Comment:
Please don't add HTML tags to documentation. Gross.
If they're needed on the output side, that ought to be handled by the documentation generator.